examples/utils/example-utils/tsconfig.json

```diff
@@ -1,9 +1,5 @@
 {
     "extends": "@fluidframework/build-common/ts-common-config.json",
```

Comment:
Should this config use `lib` for its output like the others?

Reply:
no, this config is an esnext build, it is a common js build, so it belongs in lib. I did not make it esnext only since it is consumed by other packages which they break if we do not include a commonjs version.